hack/test/travis.sh

```diff
@@ -13,13 +13,6 @@ export PATH=~/.kubeadm-dind-cluster:$PATH
 
 echo travis_fold:start:dind-up
 
-# workaround on travis: kubernetes/kubernetes-anywhere#88
```

Comment:
MEMO: I would need to do this for `/var/lib/docker/volumes/kubeadm-dind-kube-node-1/_data` if CI red